ui/shared/status/StatusEmojiPopup.qml

```diff
@@ -67,7 +67,7 @@ Popup {
         popup.close()
     }
 
-    Component.onCompleted: {
+    function populateCategories() {
```

Comment:
I tried changing the StatusEmojiPopup to a component to create it only when you click the emoji button, but it ended causing a small delay each time you clicked on it, however by changing it so it executes in `onOpened` causes the delay only the first time the popup opens (weird, i know)